tests/unit/10_build-env.bats

```diff
@@ -44,6 +44,20 @@ teardown() {
   assert_success
 }
 
+@test "(build-env) buildpack if Dockerfile present" {
```

Comment:
Fair point about the test....I'll noodle on a better approach. Happy to rebase too. Now get back to "real" work :smile: